modules/custom/az_dam/config/quickstart/media.type.ib_dam_embed.yml

```diff
@@ -0,0 +1,19 @@
+uuid: 810cdd1a-2e58-4dca-8624-b7679c64604d
```

Comment:
```suggestion
```

Reply:
Resolved locally. Will commit for review when all changes have been made.